src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1323,6 +1323,7 @@
   "com.minecolonies.coremod.furnaceuser.nofuel": "Please tell me what kind of fuel I should use in my furnace!",
   "com.minecolonies.coremod.furnaceuser.noore": "Please tell me what kind of ore I should smelt in my furnace!",
   "com.minecolonies.coremod.furnaceuser.nofood": "Please tell me what kind of food I should hand out!",
+  "com.minecolonies.coremod.sifter.nomesh": "Please make sure I have at least a string mesh to sift with!",
```

Comment:
Can probably say "a mesh to sift with" instead of "at least a string mesh to sift with", since string is the lowest level